apps/showcase/src/app/app.component.scss

```diff
@@ -1,5 +1,44 @@
+header {
+  position: sticky;
+  top: 0;
+  height: 4rem;
+  width: 100%;
+  background: var(--bs-primary);
+  box-shadow: 0 0.5rem 1rem rgba(0,0,0,0.15), inset 0 -1px 0 rgba(255,255,255,0.15);
+  z-index: 1021;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  padding-left: 1rem;
```

Comment:
should not be handle by bootstrap css class (same for margins)?

Reply:
removed